common/persistence/cassandra/cassandraPersistenceUtil.go

```diff
@@ -1054,6 +1059,7 @@ func createReplicationTasks(
 	domainID string,
 	workflowID string,
 	runID string,
+	creationTime int64,
```

Comment:
Seems like all replication task types already have the `VisibilityTimestamp` field. Can we reuse value from that field?

Reply:
The VisibilityTimestamp lives in execution table. It uses a default timestamp value for insert and select as VisibilityTimestamp is a part of the index. So I cannot use the field

Reply:
Updated all replication task creation time to use VisibilityTimestamp